### PR TITLE
Manually updating CoreCLR version packages

### PR DESCRIPTION
**Pull request by joperezr**
cc @stephentoub  @tarekgh @AlexGhiondea  @alexperovich 

For some reason the automatic PR that updates dependencies hasn't been created yet, so I'm manually updating CoreClr given that some folks depend on it and that the bug that was preventing us to consume it has been fixed.

### CODE REVIEW

File: src/Common/test-runtime/project.json [hunk 1]
```diff
@@ -1,384 +1,390 @@
 {
```
Comment (stephentoub) [inline src/Common/test-runtime/project.json]: This file looks like it contains more than version updates.  What changed here?

**Comment by tarekgh** [comment]
other than @stephentoub question, LGTM.

File: src/Common/test-runtime/project.json (hunk 1 above)
Comment (stephentoub) [inline src/Common/test-runtime/project.json]: Ah, I see.  It's more obvious with this view:
https://github.com/dotnet/corefx/pull/12805/files?w=1
Thanks

**Comment by stephentoub** [comment]
Some test failures:
```
System.Reflection.Tests.AssemblyTests.AssemblyLoadFromBytesNeg [FAIL]
  System.Runtime.Serialization.Json.Tests -> D:\j\workspace\windows_nt_de---4526f5ff\bin\Windows_NT.AnyCPU.Debug\System.Runtime.Serialization.Json.Tests\System.Runtime.Serialization.Json.Tests.dll
        Assert.Throws() Failure
        Expected: typeof(System.BadImageFormatException)
        Actual:   typeof(System.ArgumentNullException): Buffer cannot be null.
        Parameter name: buffer
        Stack Trace:
              at System.IO.MemoryStream..ctor(Byte[] buffer, Boolean writable)
              at System.Reflection.Assembly.Load(Byte[] rawAssembly, Byte[] rawSymbolStore)
              at System.Reflection.Assembly.Load(Byte[] rawAssembly)
           D:\j\workspace\windows_nt_de---4526f5ff\src\System.Runtime\tests\System\Reflection\AssemblyTests.cs(128,0): at System.Reflection.Tests.AssemblyTests.<>c.<AssemblyLoadFromBytesNeg>b__12_1()
     System.Reflection.Tests.AssemblyTests.AssemblyLoadFromBytes [FAIL]
        System.ArgumentNullException : Buffer cannot be null.
        Parameter name: buffer
        Stack Trace:
              at System.IO.MemoryStream..ctor(Byte[] buffer, Boolean writable)
              at System.Reflection.Assembly.Load(Byte[] rawAssembly, Byte[] rawSymbolStore)
              at System.Reflection.Assembly.Load(Byte[] rawAssembly)
           D:\j\workspace\windows_nt_de---4526f5ff\src\System.Runtime\tests\System\Reflection\AssemblyTests.cs(119,0): at System.Reflection.Tests.AssemblyTests.AssemblyLoadFromBytes()
```

**Comment by joperezr** [comment]
Yes, looking into those now..

**Comment by rahku** [comment]
you can disable those tests and open a bug against me . I have changed the implementation of Assembly.Load(Bytes[]..... which might be causing these. I can take a look at these failures.

**Comment by joperezr** [comment]
@rahku sounds good, I'll do that.

**Comment by joperezr** [comment]
Done. I created #12818 to track it. @rahku please take a look as soon as you can :smile:

**Comment by stephentoub** [comment]
@dotnet-bot test this please (Jenkins rebooted)

**Pull request merged**